<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizScreen1</class>
 <widget class="QWidget" name="QuizScreen1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>1201</width>
     <height>541</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-241</y>
      <width>1185</width>
      <height>780</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>780</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>989</width>
       <height>754</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="q1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Why does the 24-hour period after a storm play a crucial role in avalanche risk?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Increased atmospheric pressure</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rapid temperature fluctuations</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Snowpack overloading</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wind direction changes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q2">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>How can warm temperatures contribute to the likelihood of avalanches?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They do not contribute, they decrease risk</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By creating a slick coating of ice</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By preventing new snowfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By reducing wind speed</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q3">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is a significant factor in causing avalanches during storms?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heavy rainfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unstable snow deposits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Upward wind movement</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Lack of sunlight</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q4">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What role do varying thawing and freezing cycles play in avalanche formation?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Snow stabilization</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Weakening of upper snow layers</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Decreased atmospheric pressure</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wind direction reversal</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="quizLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Module 1 Quiz: What are the Causes of Avalanches?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>640</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Module</string>
   </property>
  </widget>
  <widget class="QLabel" name="submitMessage">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>640</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="question4"/>
  <buttongroup name="question1"/>
  <buttongroup name="question3"/>
  <buttongroup name="question2"/>
 </buttongroups>
</ui>
